### ISSUE TITLE
Collection inventory hierarchy improvements for accessbility

### ISSUE THREAD

**Issue by ggeisler** [projectblacklight/arclight#459]
The method we're currently using to produce the collection hierarchy uses `<h3>` tags for every component title, no matter where in the hierarchy the component is. In other words, every component in the hierarchy is at an `<h3>` level. So we're not providing a screenreader with a document outline that correctly represents the hierarchy of the inventory.

At some point we might want to look at this closer and see if we can improve the underlying representation of the hierarchy.

**Comment by seanaery** [comment]
Resolving this issue as part of a review of `arclight 1.0.x`. There are no longer `<h3>` tags in the hierarchy browser markup so this heading progression issue is resolved. aXe DevTools currently reports 0 accessibility issues in the hierarchy browser and it works well via keyboard nav.
